package.json

```diff
@@ -44,6 +44,7 @@
     "cookie-parser": "1.3.2",
     "copy-webpack-plugin": "3.0.1",
     "creditcards": "2.1.2",
+    "cross-env": "4.0.0-beta.0",
```

Comment:
if it's required to build the production bundles, it goes in dependencies. we do a `npm install --production` when building our prod bundles. 

usually deps are for things required to run the thing and devDeps are for things required to build the thing. In our case, it's a little muddly.